code/modules/background/space_sectors/badlands.dm

```diff
@@ -51,6 +51,12 @@
 	sector_hud_menu_sound = 'sound/effects/tajara_menu_click.ogg'
 	sector_hud_arrow = "menu_arrow"
 
+/datum/space_sector/srandmarr/get_port_travel_time()
+	return "[rand(6, 12)] hours"
```

Comment:
The port travel time here might be a bit odd on rounds where adhomai spawns, you're right near it, it's not going to be half a day until you arrive at the port, though I'm not sure how it could be made to say a much smaller number if adhomai is spawned...

Reply:
You still need to hitch a shuttle to get to Adhomai, which takes time. These numbers were approved by Alb